lib/process-services/src/lib/task-list/components/task-details.component.ts

```diff
@@ -395,6 +396,24 @@ export class TaskDetailsComponent implements OnInit, OnChanges, OnDestroy {
         this.loadDetails(taskId);
     }
 
+    onTaskClaim(taskClaimStatus: TaskClaimModel): void {
+        if (taskClaimStatus.status === ClaimStatusType.CLAIM) {
+            this.claimedTask.emit(taskClaimStatus.taskId);
+            this.taskFormComponent.loadTask(taskClaimStatus.taskId);
+            this.loadDetails(taskClaimStatus.taskId);
+        }
+
+        if (taskClaimStatus.status === ClaimStatusType.UNCLAIM) {
+            this.unClaimedTask.emit(taskClaimStatus.taskId);
+            this.taskFormComponent.loadTask(taskClaimStatus.taskId);
+            this.loadDetails(taskClaimStatus.taskId);
+        }
+
+        if (taskClaimStatus.status === ClaimStatusType.FAILED) {
```

Comment:
the error should not be strictly related to the claim, it should be generic. 
When the complete fail, when the claim fails, when a custom outcome fails